doc/project_governance/rfc_approved.md

```diff
@@ -0,0 +1,8 @@
+# OpenTitan RFCs
+
+The [OpenTitan RFC \(request for comment\)](./rfc_process.md) process is intended to provide a consistent and controlled path for changes (such as new features) so that all stakeholders can be confident about the direction of the project.
+
+RFCs indicate the significant changes on OpenTitan which have recently been agreed or are being discussed **right now** and can be viewed at the links below:
+- [Approved RFCs](https://github.com/lowRISC/opentitan/issues?q=is%3Aissue%20label%3ARFC%3AApproved)
```

Comment:
These links are heavily outdated. Most of the RFCs are in private. Also these issues often point to private documents.